venues/EMNLP/python/init.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python
+
+import argparse
+import openreview
+import datetime
+
+"""
+OPTIONAL SCRIPT ARGUMENTS
+
+    baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+    username - the email address of the logging in user
+    password - the user's password
+"""
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print('connecting to {0}'.format(client.baseurl))
+
+emnlp_request_id = 'rylllwkbKN'
+conference = openreview.helpers.get_conference(client, emnlp_request_id)
+
+author_consent_agreement = (
+    '**IMPORTANT: PLEASE READ**\n\n'
+    'The workshop is collecting a research dataset containing a set of peer'
+    'reviews for research purposes. This dataset may be released to the '
+    'public domain six months after the final accept/reject decisions are made. '
+    'Do you agree to include your reviews in this dataset?'
+)
+
+additional_fields = {
+    "consent": {
+        "value-radio": [
+                "Yes, include my anonymized reviews in the \"workshop research dataset\".",
+                "No. Do not include my reviews in the \"workshop research dataset\"."
+        ],
+        "default": "Yes, include my anonymized reviews in the \"workshop research dataset\".",
+        "description": author_consent_agreement,
+        "scroll": True
+    }
+}
+
+conference.open_submissions(additional_fields=additional_fields)
```

Comment:
don't we need to set the start and due date from the request form here?

Reply:
I didn't have a great understanding of how the builder worked when I wrote this, but I just looked at the code, and you're right: it does need the dates. I'll fix that.

At the time, I was thinking that `open_submissions` would know about the conference's current state after the `get_conference` call. I thought that I would only need to add information that the request form didn't already have.

One thing I did notice: it looks like `get_conference` is calling `open_submissions` itself. It was surprising to me that a function that has "get" in the name could potentially change the state of the conference (it didn't in this case, because the submissions were already open, so it works fine here). Just thought I would mention it, as a person who is new to the builder/conference API. It could cause trouble in the future.

Reply:
get_conference gives you an instance of the Conference object with the status reflected in the request note. 

To be sure the objects in the database are synced with the request note we need to call to update all the groups and invitations, otherwise we need to do a diff to check if something has changed. 

```open_submissions``` is not a good name, we should rename it in the future. 

If you want to overwrite something in the invitation you need to call ```open_submissions``` with all the parameters, we currently don't have a method to update the additional fields only.